.github/workflows/test.yml

```diff
@@ -75,15 +75,15 @@ jobs:
     steps:
```

Comment:
```suggestion
    steps:
        - uses: alexellis/setup-arkade@master
        - name: Install vmmeter
           run: |
            sudo -E arkade oci install ghcr.io/openfaasltd/vmmeter:latest --path /usr/local/bin/
        - uses: self-actuated/vmmeter-action@master
```

Reply:
Seems like the e2e tests didn't run at all after this change?